src/solvers/strings/string_refinement.cpp

```diff
@@ -988,12 +988,12 @@ static optionalt<exprt> get_valid_array_size(
     stream << "(sr::get_valid_array_size) size is not valid" << messaget::eom;
     return {};
   }
-  std::size_t n = *num_size_opt;
+  std::size_t num_size = *num_size_opt;
```

Comment:
:pick: I would rather name this one `size` and the previous size `size_expr`

Reply:
Got rid of that change